bvm/ballerina-core/src/main/resources/MessagesBundle.properties

```diff
@@ -161,3 +161,4 @@ invalid.object.field.addition = invalid value for object field ''{0}'': expected
 dynamically.nested.transactions.are.not.allowed = dynamically nested transactions are not allowed
 cyclic.value.reference = ''{0}'' value has cyclic reference
 tuple.index.out.of.range = tuple index out of range: index: {0}, size: {1}
+illegal.array.insertion= cannot insert an element into index ''{0}'' of an array without filler values
```

Comment:
Not sure if this message conveys what we want to. The index alone doesn't matter here right? Since the issue arises based on a combination of the current size and the index?

Should we include something that indicates this is related to expanding the array?